torch/lib/c10d/Utils.hpp

```diff
@@ -195,7 +195,7 @@ inline void assertTypeAndSizesMatch(
   assertTypeAndSizesMatch(fn, tensors.slice(1), type, sizes);
 }
 
-// Copied from torch/csrc/utils/functional.h.
+// Copied from ATen/core/functional.h.
```

Comment:
I think it could even be deleted now that it can use the c10 version directly. The copy was done originally to avoid including straight from csrc where all other includes were aten/c10. Feel free to ignore though, this is not critical.

Reply:
It doesn't look like functional.h is moved to c10 yet.